### PR TITLE
POC bug fixes: support multi-org, multi-roles project.yml

### PR DESCRIPTION
**Pull request by chesterxgchen**
1. fix a bug in cli.py
2. make sure poc command handling multi-org, multi-roles cases
3. update poc tutorials to show multi-org, multi-roles admin console start and stop
4. fix test cases
5. change the syntax ```nvflare poc prepare-examples -e <examples_dir>``` to ```nvflare poc prepare-jobs-dir -j <jobs_dir>``` considering that examples just example directory for different jobs. 

Fixes # .

### Description

A few sentences describing the changes proposed in this pull request.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/tool/poc/poc_commands.py [hunk 1]
```diff
@@ -218,10 +221,21 @@ def get_fl_server_name(project_config: OrderedDict) -> str:
...
+
+
+def get_other_admins(project_config: OrderedDict):
+    return get_fl_admins(project_config, is_project_admin=False)
 
+
+def get_proj_admin(project_config: OrderedDict):
+    admins = get_fl_admins(project_config, is_project_admin=True)
     if len(admins) == 1:
         return admins[0]
```
Comment (YuanTingHsieh) [inline nvflare/tool/poc/poc_commands.py:240]: we can only have 1 project admins? @IsaacYangSLA

File: nvflare/lighter/service_constants.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 class FlareServiceConstants(object):
     FLARE_PROJ_ADMIN = "admin@nvidia.com"
+    FLARE_OTHER_ADMINS = "other_admins"
     FLARE_SERVER = "server"
     FLARE_CLIENTS = "clients"
     FLARE_OVERSEER = "overseer"
```
Comment (YuanTingHsieh) [inline nvflare/lighter/service_constants.py:21]: if these constants only used by "tool/poc/" should we move them there?

File: examples/tutorials/setup_poc.ipynb [hunk 3]
```diff
@@ -429,15 +462,75 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f4f80e58-92d2-4dca-be1a-a909ede39fca",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
```
Comment (YuanTingHsieh) [inline examples/tutorials/setup_poc.ipynb:525]: For unknown reasons, this notebook file can not be rendered on GitHub website, maybe this part needs to be removed?

File: nvflare/tool/poc/poc_commands.py (hunk 1 above)
Comment (YuanTingHsieh) [reply nvflare/tool/poc/poc_commands.py:240]: the else condition say:
```
else:
    raise CLIException(f"project should only have only one project admin, but {len(admins)} are provided: {admins}")
``` 
        
 ^ is this statement true?

Comment (yhwen) [reply nvflare/tool/poc/poc_commands.py:240]: There should be only ONE project_admin allowed through the provision tool. @IsaacYangSLA will add the enforcement in the future.

File: nvflare/cli.py [hunk 4]
```diff
@@ -193,6 +194,7 @@ def print_help(prog_parser, sub_cmd, sub_cmd_parsers):
     if sub_cmd:
         sub_parser = sub_cmd_parsers[sub_cmd]
         if sub_parser:
+            print(sub_parser)
```
[the remark this answers is not in the document]
Comment (yhwen) [reply nvflare/cli.py]: Updated the print out message. (useful information.)

File: examples/tutorials/setup_poc.ipynb (hunk 3 above)
Comment (yhwen) [reply examples/tutorials/setup_poc.ipynb:525]: I don't have the issue for GitHub rendering this file. Maybe it's just a one time GitHub issue you experienced.

Comment (YuanTingHsieh) [reply examples/tutorials/setup_poc.ipynb:525]: main branch rendered. this file does not.
I am using Firefox with Ubuntu OS

File: nvflare/cli.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/cli.py]: @yhwen should we just remove this print?

File: examples/tutorials/setup_poc.ipynb (hunk 3 above)
Comment (yhwen) [reply examples/tutorials/setup_poc.ipynb:525]: I don't have issue for reentering the setup_poc.ipynb file, Firefox + Ubuntu
also works on Mac

**Comment by IsaacYangSLA** [comment]
notebook rendering issue seemed not existing in local machine.  I think we can merge it.

**Review by IsaacYangSLA — APPROVED** [review]
Thanks.  Some cells not rendered properly in the notebook.  Other than that, it looks good to me.

**Pull request merged**